test/action.yml

```diff
@@ -0,0 +1,57 @@
+---
+name: Test Dotnet
+description: |
+  Github Action to Build & Test Dotnet
+
+  RequiredEnv:
+    GITHUB_TOKEN
+    SONAR_TOKEN
+inputs:
+  dotnet-version:
+    description: >
+      The dotnet-version input is optional.
+      The default version of Dotnet in PATH varies
+      between runners and can be changed unexpectedly
+      so we recommend always setting Dotnet version explicitly
+      using the dotnet-version input.
+    required: false
+runs:
+  using: composite
+  steps:
+    - name: Set up Dotnet
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: ${{ inputs.dotnet-version }}
+
+    - name: Set up Dotnet
```

Comment:
How com we are setting fixed for version 5.0.x and one from inputs?

Reply:
dotnet sonarscans cli requires 5.0.x to run so installing on top of application runtime